test/schema.test.ts

```diff
@@ -32,14 +36,18 @@ describe("commands/schema", function () {
 
   test("throws with missing table", async function () {
     const consoleError = spy(console, "error");
-    await yargs(["schema", "ignored_31337_99"]).command(mod).parse();
+    await yargs(["schema", "ignored_31337_99", "--chain", "local-tableland"])
+      .command(mod)
+      .parse();
     assert.calledWith(consoleError, "Not Found");
   });
 
   test("Schema passes with local-tableland", async function () {
-    const consoleLog = spy(console, "log");
-    await yargs(["schema", "healthbot_31337_1"]).command(mod).parse();
-    assert.calledWith(consoleLog, {
+    const consoleDir = spy(console, "dir");
+    await yargs(["schema", "healthbot_31337_1", "--chain", "local-tableland"])
```

Comment:
Shouldn't chain still be optional here?

Reply:
You are correct, it's optional, and no longer specified in the test in the newest version of this file.